swift/ios-application/ios-plugin/src/main/groovy/org/gradle/samples/plugins/IOSApplicationPlugin.groovy

```diff
@@ -0,0 +1,94 @@
+package org.gradle.samples.plugins
```

Comment:
why do we use Kotlin for the build scripts and Groovy for the plugin implementation?

Reply:
I think we should avoid using Groovy for sample plugins. We don't suggest that people to use Groovy as an implementation language for their plugins. Currently, we recommend using Java but it would be fine to also have a Kotlin implementation.

Reply:
I have open issue https://github.com/gradle/gradle-native/issues/655 to track this discussion.